cypress/e2e/errorHandling_spec.cy.ts

```diff
@@ -0,0 +1,41 @@
+describe('Error handling', () => {
+
+  it('should show an error image when there is a 500 error', () => {
+    cy.intercept({url:'https://api.thedogapi.com/v1/breeds'}, {statusCode: 500})
+    cy.visit('http://localhost:3000/')
+    cy.get('.error-message')
+      .contains('500:function text() { [native code] }')
+    cy.get('.error-image').should('be.visible')
+  })
+})
+
+describe('Error handling', () => {
```

Comment:
Do each of these tests need to be in their own Describe block? They seem similar that they could be grouped together under one Describe block with multiple it blocks.